### PR TITLE
update: new Falco Prow 

### PR DESCRIPTION
**Pull request by fntlnz**
Authors: @jonahjon and @maxgio92 - It's under my name because we did a rebase in the Infra WG.

MVP for the prow microservice compenents.

Includes

1 cluster prow setup, without github pieces finished.
IRSA through terraform for s3 accessing microservices
Updated deployment scripts and makefile for easy up/down
New region (eu-west-1) and instance type (m5.large) from last community call.
Branding setup for Prow Deck working properly.

WIP
Start of the 1 password scripts for CircleCI automation in the future

Screen Shot 2020-10-14 at 6 16 09 PM

Supersedes #197 
Originally created by @maxgio92 and @jonahjon

### CODE REVIEW

**Comment by nibalizer** [comment]
Note credit for this PR goes to @jonahjon and @maxgio92, this PR is just rebased and resubmitted by @fntlnz

**Comment by fntlnz** [comment]
Thanks @nibalizer - I added a note at the very top - All the credit goes to them!

**Comment by fntlnz** [comment]
In the community call today we discussed about merging this before the new infra is available.

We need to check if there are any conflicts that can happen with the actual prow installation since it has a mechanism to auto update when the configuration files change.

**Comment by jonahjon** [comment]
> In the community call today we discussed about merging this before the new infra is available.
> 
> We need to check if there are any conflicts that can happen with the actual prow installation since it has a mechanism to auto update when the configuration files change.

Took a look at the configs and plugins files..... for the time being if we want to merge this we NEED to make sure the original `plugins.yaml` and `configs.yaml` stay in place.

In the `plugins.yaml` the config_updater is looking for both those config paths, that if we delete the original that could be destructive.  So we create the new file's, but don't delete the original yet.

```
config_updater:
  config_file: prow/config/configs.yaml
  plugins_file: prow/config/plugins.yaml
  maps:
    prow/config/configs.yaml:
      name: configs
    prow/config/plugins.yaml:
      name: plugins
```

File: config/clusters/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+Default region name [None]: <YOUR_AWS_REGION>
+Default output format [None]: json
+```
+
+This enables Terraform access to the configuration file and performs operations on your behalf with these security credentials.
+
+Copy the environment var file from the sample `demo.tfvars.sample`:
+
+```shell
+cp demo.tfvars.sample demo.tfvars
```
Comment (fntlnz) [inline config/clusters/README.md:21]:
```suggestion
cp prow.tfvars.sample prow.tfvars
```

File: config/clusters/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+$ aws configure
+AWS Access Key ID [None]: <YOUR_AWS_ACCESS_KEY_ID>
+AWS Secret Access Key [None]: <YOUR_AWS_SECRET_ACCESS_KEY>
+Default region name [None]: <YOUR_AWS_REGION>
+Default output format [None]: json
+```
+
+This enables Terraform access to the configuration file and performs operations on your behalf with these security credentials.
+
+Copy the environment var file from the sample `demo.tfvars.sample`:
```
Comment (fntlnz) [inline config/clusters/README.md:18]:
```suggestion
Copy the environment var file from the sample `prow.tfvars.sample`:
```

**Comment by maxgio92** [comment]
thanks @fntlnz I committed your suggestions and put you as co-author

**Comment by jonahjon** [comment]
> @jonahjon should this name be fixed? I'm just wondering if it can scale


There is actually only 1 bucket per prow setup no matter the amount of worker clusters. The worker cluster all stored get stored in the control plane's single bucket.

**Comment by leodido** [comment]
Can we close this in favor of #204?

@falcosecurity/test-infra-maintainers

**Pull request closed without merging**